gallery/src/components/NxTextLink/NxTextLinkInternalExample.tsx

```diff
@@ -30,14 +30,14 @@ export default () =>
       </NxCode>
     </NxP>
     <NxP>
-      <NxTextLink href="#/pages/nx-code">
+      <NxTextLink href="#/pages/Code">
         <NxCode>
           NxCode inside NxTextLink (Not Recommended)
         </NxCode>
       </NxTextLink>
     </NxP>
     <NxP>
-      <NxTextLink href="#/pages/nx-code">
+      <NxTextLink href="#/pages/Code">
```

Comment:
both pages linking to code brings you to mid-page as well

Reply:
I found this issue as well and created a ticket for it below:
https://issues.sonatype.org/browse/RSC-1089

Since this issue should not be fixed during this ticket. I'm closing this conversation.